build.gradle.kts

```diff
@@ -1,42 +1,45 @@
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
-    id("org.jetbrains.intellij") version "0.4.21"
+    id("org.jetbrains.intellij") version "0.5.0"
     java
-    kotlin("jvm") version "1.3.72"
+    kotlin("jvm") version "1.4.10"
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
 }
 
 tasks.withType<KotlinCompile>().configureEach {
     kotlinOptions.jvmTarget = "11"
 }
 
 group = "de.tum.www1.artemis.plugin.intellij"
-version = "1.0.1"
+version = "2.0"
```

Comment:
I wanted to make sure the students see the big new version and upgrade the plugin to avoid the crash. But I see you want semantic versioning, will change it.